src/test/java/lotto/domain/LottoResultTest.java

```diff
@@ -13,38 +13,43 @@
 public class LottoResultTest {
     LottoResult result;
     Lottos lottos;
+    LottoNumber lottoNumber;
```

Comment:
```suggestion
    LottoNumber bunusNumber;
```
더 명확한 이름을 사용하는게 좋을것같아요!

Reply:
네~ 명확한 이름으로 수정하였습니다 👍👍